frontend/public/components/RBAC/bindings.jsx

```diff
@@ -283,13 +283,7 @@ export const RoleBindingsPage = ({
   showTitle = true,
   mock = false,
   staticFilters = undefined,
-  name,
-  kind,
```

Comment:
Do we no longer need `name` and `kind`?

Reply:
Whoops.  Good catch, @spadgett.  I omitted the fixes from #7546 and #7581 that were using `name` and `kind` and didn't notice.  Updated.